ext/pdo_sqlite/php_pdo_sqlite_int.h

```diff
@@ -76,7 +76,8 @@ extern const struct pdo_stmt_methods sqlite_stmt_methods;
 
 enum {
 	PDO_SQLITE_ATTR_OPEN_FLAGS = PDO_ATTR_DRIVER_SPECIFIC,
-	PDO_SQLITE_ATTR_READONLY_STATEMENT
+	PDO_SQLITE_ATTR_READONLY_STATEMENT,
+	PDO_SQLITE_ATTR_EXTENDED_RESULT_CODES
```

Comment:
This value needs to be registered here in the `PHP_MINIT_FUNCTION(pdo_sqlite)` function to make the `PDO::SQLITE_ATTR_EXTENDED_RESULT_CODES`  class constant available